site/docs/hosting/vps.md

```diff
@@ -45,131 +137,638 @@ cd /etc/systemd/system
 2. Open your new service file with an editor:
 
 ```sh
-nano bot1.service
+nano <app-name>.service
 ```
 
+replacing `<app-name>` with any identifier.
+
+> `<app-name>.service` will be the name of your service.
+
 3. Add the following content:
 
 ```text
+[Unit]
+After=network.target
+
 [Service]
-ExecStart=<start_command>
+Environment=BOT_TOKEN=<token>
+ExecStart=<start-command>
 Restart=on-failure
 
 [Install]
 WantedBy=multi-user.target
 ```
 
-> Replace `<start_command>` with the command you got above.
->
-> Also note that if Deno is installed for a different user other than root, you may need to specify it in the service section like `User=<the_user>`.
-> For more information on the unit files, visit [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_working-with-systemd-unit-files_configuring-basic-system-settings).
+replacing `<token>` with your bot's token, `<launch-command>` with the command you received [above](#getting-the-start-command), and `<username>` with the name of the user on whose behalf the app is being started.
 
-3. Reload systemd whenever you edit the service:
+A small explanation of the service configuration:
 
-```sh
+- `After=network.target` - indicates that the application should be launched after the Internet module is loaded.
+- `Environment=BOT_TOKEN=<token>` - sets the environment variable.
+  Add new `Environment` if you need multiple environment variables.
+- `ExecStart=<start-command>` - sets the startup command.
+- `Restart=on-failure` - indicates that the application should restart after a crash.
+- `WantedBy=multi-user.target` - defines the system state in which the service should be launched.
+  `multi-user.target` - is a typical value for servers that defines the state of the system when everything works except the graphical interface, which simply does not exist.
```

Comment:
I'm not sure if this is correct, because I'm not very good with systemd

Reply:
I don't know, either, so let's just leave it like that until someone complains.